chapters/ja/_toctree.yml

```diff
@@ -6,4 +6,9 @@
 - title: 1. Transformerモデルについて
   sections:
   - local: chapter1/1
-    title: イントロダクション
\ No newline at end of file
+    title: イントロダクション
+
+- title: Hugging Faceコースのイベント
+  sections:
+  - local: event/1
+    title: Chapter2公開記念イベント
```

Comment:
It's been some time since I studied Japanese, but in the English version we refer to this as "Part 2 Release Event", so should the translation be something like

> 第2部発売記念イベント

I'm just asking why you chose to replace "Part 2" with "Chapter 2" :) Just FYI we use "Part 2" to refer to chapters 4-8

Reply:
I second @lewtun , to make it consistent with the other PR (https://github.com/huggingface/course/pull/244) I think you should put `パート2` and not `Chapter2`

Reply:
Thanks for the confirmation.

「パート2公開記念イベント」
is a good idea?

I had thought Part2 mean is [this repository's chapter2](https://github.com/huggingface/course/tree/main/chapters/en/chapter2).
However, I should put PART2 to make it consistent with PR (#244) as @younesbelkada said.

And, I think "Release" should be translated to "公開" because I predict this "Release" means "publish".
"発売" means "sale".

Reply:
Perfect thank you very much! Agreed with you

Reply:
"パート2公開記念イベント" sounds good to me too!

Reply:
I fixed in [6075579](https://github.com/huggingface/course/pull/241/commits/60755792eb0ea244cd7f919b41aa011edde851e9)

Thanks!